contracts/mirror_mint/src/asserts.rs

```diff
@@ -57,10 +62,11 @@ pub fn assert_auction_discount(auction_discount: Decimal) -> StdResult<()> {
 }
 
 pub fn assert_min_collateral_ratio(min_collateral_ratio: Decimal) -> StdResult<()> {
```

Comment:
But we already have two assets with auction `discount > min_collateral_ratio - 1`. So, even if we modify the assertion to not allow new assets to have such lows minCR, we still have to handle the two assets that have 110% minCR. Because of this, I think we need to keep the other changes.

Reply:
Ah ok, understood